[2024-05-04 10:48:20,914] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_2 2024-05-04T10:41:16.770636+00:00 [queued]>
[2024-05-04 10:48:20,934] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_2 2024-05-04T10:41:16.770636+00:00 [queued]>
[2024-05-04 10:48:20,935] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-05-04 10:48:20,935] {taskinstance.py:1043} INFO - Starting attempt 2 of 2
[2024-05-04 10:48:20,936] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-05-04 10:48:20,949] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): runme_2> on 2024-05-04T10:41:16.770636+00:00
[2024-05-04 10:48:20,953] {standard_task_runner.py:52} INFO - Started process 1732 to run task
[2024-05-04 10:48:20,957] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'runme_2', '2024-05-04T10:41:16.770636+00:00', '--job-id', '38', '--pool', 'default_pool', '--raw', '--subdir', '/home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpa5uoy_zh', '--error-file', '/tmp/tmprqt4yczd']
[2024-05-04 10:48:20,960] {standard_task_runner.py:77} INFO - Job 38: Subtask runme_2
[2024-05-04 10:48:21,007] {logging_mixin.py:104} INFO - Running <TaskInstance: example_bash_operator.runme_2 2024-05-04T10:41:16.770636+00:00 [running]> on host 76f9873b9902
[2024-05-04 10:48:21,060] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=runme_2
AIRFLOW_CTX_EXECUTION_DATE=2024-05-04T10:41:16.770636+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-05-04T10:41:16.770636+00:00
[2024-05-04 10:48:21,062] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2024-05-04 10:48:21,064] {bash.py:158} INFO - Running command: echo "example_bash_operator__runme_2__20240504" && sleep 1
[2024-05-04 10:48:21,072] {bash.py:169} INFO - Output:
[2024-05-04 10:48:21,076] {bash.py:173} INFO - example_bash_operator__runme_2__20240504
[2024-05-04 10:48:22,081] {bash.py:177} INFO - Command exited with return code 0
[2024-05-04 10:48:22,121] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=runme_2, execution_date=20240504T104116, start_date=20240504T104820, end_date=20240504T104822
[2024-05-04 10:48:22,189] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-04 10:48:22,216] {local_task_job.py:146} INFO - Task exited with return code 0
